.github/workflows/validate-migration.yaml

```diff
@@ -0,0 +1,101 @@
+name: validate-migration
+
+on: [ push ]
+
+env:
+  # Constants
+  P4_DOWNLOAD_URL: "https://www.perforce.com/downloads/perforce/r23.1/bin.linux26x86_64/p4api-glibc2.3-openssl1.0.2.tgz"
+  OPENSSL_DOWNLOAD_URL: "https://www.openssl.org/source/openssl-1.0.2u.tar.gz"
+  OPENSSL_INSTALL_DIR: "/tmp/openssl-install"
+
+  # Perforce credentials
+  P4PORT: "perforce.sgdev.org:1666" # the address of the Perforce server to connect to
+  P4USER: "admin" # the name of the Perforce user
+  P4PASSWD: ${{ secrets.P4PASSWD }} # the ticket for the Perforce user
+
+  DEPOT_NAME: "source/src-cli" # the name of the Perforce depot to validate the git migration against
+
+jobs:
+  validate-migration:
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer:
+          - addressSanitizer
+          - threadSanitizer
+          #          disabled until we figure out how to build with clang in github actions
+          #          - memorySanitizer
+          - leakSanitizer
```

Comment:
ASan enables LSan by default, so this is unnecessary. https://clang.llvm.org/docs/LeakSanitizer.html